egg-herbie/src/lib.rs

```diff
@@ -317,7 +317,7 @@ pub unsafe extern "C" fn egraph_get_proof(
 
         let mut proof = runner.explain_equivalence(&expr_rec, &goal_rec);
         ctx.runner = Some(runner);
-        let string = CString::new(proof.get_flat_string()).unwrap();
+        let string = CString::new(proof.get_string_with_let().replace("\n", "")).unwrap();
```

Comment:
Is the replace necessary?

Reply:
It's a hack because we delimit different results from egg with newlines.
Not too motivated to fix it, mostly because I'm trying to replace this code anyways in my branch

Reply:
Ugly but ok